### ISSUE TITLE
Search for MED,23589462 does not work

### ISSUE THREAD

**Issue by dahoo** [citenet/citenet#11]
For some reason the search for ID "MED,23589462" returns a dict without this ID, which causes an error.

**Comment by jorenretel** [comment]
I am trying to recreate the bug. On which branch are you now?
Because I just pulled and it seems like the master is in the state like it is on heroku: not allowing to fill out the document ID.

**Comment by dahoo** [comment]
I'm on the master branch. I enabled the search again.

**Comment by jorenretel** [comment]
Do you still have the bug? Because I am not able to reproduce it. Everything seems to work.

**Comment by dahoo** [comment]
Yes, I still have it. I get `script.js:69: Uncaught TypeError: Cannot read property 'title' of undefined`.

**Comment by jorenretel** [comment]
I can still not reproduce the bug. I also checked whether the ID "MED,23589462" is really in the python dict and it is. What is the difference between your set up and mine? I am testing this on the master branch and it seems like all files are tracked so that can not be the difference.
Did you change something on the master that you did not push? Is there any difference between your local master and the remote master?: git diff master origin/master
So this id is the standard one we are using. What about another ID, like: MED,25111632 ?

**Comment by dahoo** [comment]
Ahh, the difference is the crawl depth. I used 20 and it's 10 on master.

**Comment by jorenretel** [comment]
But that is still a bug! Okay, I know where the bug is. It is in the post filter. It throws out this paper because it has only one citation in the local network.

**Comment by jorenretel** [comment]
I will solve it. I think I already solved it in my reorganize branch, but forgot to put it on master. Wait a second..

**Comment by jorenretel** [comment]
I pushed the bug fix. Sorry for that one.

**Comment by jorenretel** [comment]
By the way, in branch 'reorganize' I reorganized the crawler. Could you take a look at it? And tell me what you think? I want to make a few more changes, and then I will open a pull request, because it changes quite a bit of the code we wrote during the weekend.

**Comment by dahoo** [comment]
Thanks, but now there are papers without connections:
![image](https://cloud.githubusercontent.com/assets/1329019/4844007/0be22bfe-6038-11e4-913f-709dc0ba3ef2.png)

I'll look into your changes!

**Comment by jorenretel** [comment]
I know why. The post filter throws out too many articles. I'll fix it when I get back to my computer. It's simple, there needs to be a rule that papers that are linking to a paper that is displayed should never be thrown out.

----- Reply message -----

**Comment by jorenretel** [comment]
The filter is basically too aggressive.
